pyproject.toml

```diff
@@ -59,6 +59,9 @@ all_models=[
 ]
 all=["PVNet[dev,all_models]"]
 
+[tool.isort]
+profile = "black"
+
```

Comment:
If we're moving to ruff, `isort` can go altogether - ruff handles sorting by itself.